help_to_heat/frontdoor/routing/backwards_routing.py

```diff
@@ -15,6 +15,21 @@
 
 
 def get_prev_page(current_page, answers):
+    """
+    Generates the previous page ID in the flow, given the users' current page and answers.
+    Parameters
+    ----------
+    current_page
+        Page ID the user is currently on.
+    answers
+        All answers given by the user so far. Should include answers submitted on the current page
+
+    Returns
+    -------
+    str | None
```

Comment:
```suggestion
    str
```
and presumably in the other place you copied this from if that's where it's come from?